### PR TITLE
Added -s flag for set position by char seq '^&'.

### PR DESCRIPTION
**Pull request by chu4ng**
#### ISSUE TYPE

- Improvement/feature implementation

#### RUNTIME ENVIRONMENT

- Operating system and version: Archlinux, 5.10.8-arch1-1
- Terminal emulator and version: st
- Python version: 3.9.1
- Ranger version/commit: ranger-master v1.9.3-174-g290c1f5f
- Locale: en_US.UTF-8

#### CHECKLIST

- [X] The `CONTRIBUTING` document has been read **[REQUIRED]**
- [X] All changes follow the code style **[REQUIRED]**
- [X] All new and existing tests pass **[REQUIRED]**
- [X] Changes require config files to be updated (config/command.py)

#### DESCRIPTION

Added '-s' flag for "console" command. It sets cursor position at index of special characters.
'%^' make some errors and '^&' look pretty and little logical.

#### MOTIVATION AND CONTEXT

#1914
And it's actully usefull feature.

### CODE REVIEW

**Comment by toonn** [comment]
Hmm, maybe the first argument should be the sentinel character? The same way vim's substitute can use slashes or colons depending on the first separator, e.g., `s/pattern/replacement/` and `s:pattern:replacement:`.

**Comment by chu4ng** [comment]
@toonn Ok, it looks logical. But I'm not sure about error handler, if someone will miss this argument?

**Comment by toonn** [comment]
Just weird behavior probably. But better than being forced to escape whichever random character we choose.

File: ranger/config/commands.py [hunk 1]
```diff
@@ -828,21 +828,32 @@ def execute(self):
 
 
 class console(Command):
-    """:console <command>
+    """:console [-p N] [-s sep] <command>
```
Comment (toonn) [inline ranger/config/commands.py:831]: Maybe `[-p N | -s sep]`, to make it clear you can't use both?

**Review by toonn — COMMENTED** [review]
Looks good to me but I have to get around to testing.

**Comment by toonn** [comment]
Great contribution! 🎉

**Comment by chu4ng** [comment]
Thank ^^

**Pull request merged**